easybuild/easyconfigs/n/ncview/ncview-2.1.7-gompi-2020a.eb

```diff
@@ -0,0 +1,42 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+##
+easyblock = 'ConfigureMake'
+
+name = 'ncview'
+version = '2.1.7'
```

Comment:
bump to 2.1.8 (just writing a review so that github realized there are changes requested)

Reply:
bumped to 2.1.8 as requested